app/views/videos/video.html.haml

```diff
@@ -15,34 +15,30 @@
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form
+      = bootstrap_form_for [@video, Review.new] do |f|
         %fieldset
           .form-group
-            %label Rate this video
             .row
               .col-sm-3
-                %select.form-control(name="")
-                  %option(value="5") 5 Stars
-                  %option(value="4") 4 Stars
-                  %option(value="3") 3 Stars
-                  %option(value="2") 2 Stars
-                  %option(value="1") 1 Star
+                = f.select :rating, [["1 Star", 1], ["2 Stars", 2], ["3 Stars", 3], ["4 Stars", 4], ["5 Stars", 5]], {label: "Rate this video"}
           .form-group
-            %label Write Review
             .row
               .col-sm-8
-                %textarea.form-control(name="" rows="6")
+                = f.text_area :body, placeholder: 'Leave a review here', label: 'Write Review', rows: '6'
+          .form-group
+            = f.text_field :user_id, value: current_user.id, type: 'hidden', hide_label: true
```

Comment:
This is not a good idea. You have access to current_user in your controller and here you only expose it to malicious users.